deps/common/src/logseq/common/path_test.cljs

```diff
@@ -0,0 +1,27 @@
+(ns logseq.common.path-test
```

Comment:
Could you add tests for more of the main fns? Perhaps some of the deleted tests of `capacitor-fs-test` can be reused. I'd be hesitant to use this library going forward if there's little shared understanding of what the expected behavior is. I can help with desktop tests if you'd like